common/src/main/java/com/microsoft/hydralab/common/file/ContainerConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.microsoft.hydralab.common.file;
+
+public interface ContainerConstants {
```

Comment:
If it's constants, shouldn't it only contain constants? If it's a getter collection, a better name is StorageProperties

Reply:
Resolved.